modules/tinymce/src/core/main/ts/api/dom/ControlSelection.ts

```diff
@@ -201,23 +203,26 @@ const ControlSelection = (selection: Selection, editor: Editor): ControlSelectio
     }
   };
 
-  const endGhostResize = function () {
+  const endGhostResize = () => {
+    const wasResizedStarted = resizeStarted;
```

Comment:
typo? "resized"

Reply:
Haha, oops. I originally called this `wasResized` but realised that wasn't correct and added `Started`. Seems I missed removing the "d", so thanks for the catch!